flake8_trio/visitors/visitor2xx.py

```diff
@@ -190,7 +111,7 @@ def visit_blocking_call(self, node: ast.Call):
 
 
 @error_class
-class Visitor212(TypeTrackerVisitor200):
+class Visitor212(Visitor200):
```

Comment:
Now inherits from 200 like the rest of the 2xx so `visit_blocking_call` is called, and variable tracking is now shared across all runners.